### PR TITLE
Remove sustain

### PR DESCRIPTION
**Pull request by rwwagner90**
Removed the sustain stuff and pulling it in via the new flexi-sustain package.

Think it all works as it should. Please leave comments and check everything out 😄

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -38,6 +38,7 @@
     "ember-load-initializers": "^0.5.1",
     "ember-resolver": "^2.0.3",
     "ember-suave": "^4.0.0",
+    "flexi-sustain": "1.2.0",
```
Comment (kybishop) [inline package.json]: Artifact of testing?

Comment (rwwagner90) [inline package.json]: Yeah, just in `devDependencies` for testing purposes

**Comment by rwwagner90** [comment]
@kybishop isn't the whole point of yarn to check in `yarn.lock` to ensure all the packages are on the same version between all developers working on the project? May or may not want it here, but could be helpful to ensure we're all on a consistent version of everything.

**Comment by kybishop** [comment]
@rwwagner90 I'm game for committing to yarn.

File: addon/services/-sustains.js [hunk 2]
```diff
@@ -1,60 +0,0 @@
-import Ember from 'ember';
-import SustainModel from '../lib/sustain';
-import getOwner from 'ember-getowner-polyfill';
```
Comment (kybishop) [inline addon/services/-sustains.js]: This is the only time we use `ember-getowner-polyfill`. Safe to remove from package.json now.

Comment (rwwagner90) [inline addon/services/-sustains.js]: Good catch, just removed it!

File: package.json (hunk 1 above)
Comment (runspired) [inline package.json]: What are the things that keep us from testing the sustain in it's own repo and just pulling in the other parts of flexi needed to do so as a dev-dependency there?

**Pull request merged**